packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -63,6 +63,16 @@ export interface TextInputProps
    */
   placeholder?: string;
 
+  /**
+   * An optional prefix displayed before the input
+   */
+  prefix?: string;
```

Comment:
We'll likely want this to be `ReactNode` or similar, since users can pass in strings, xIcon, or Button.